resources/entrypoint.sh

```diff
@@ -23,5 +23,12 @@ if [ "$EXIT" -eq 1 ]; then
     exit 1
 fi
 
+# We need a host to send our stats. I happen to know that our collector is
```

Comment:
Could we change this comment to something along these lines?

```
# By default, set STATSD_HOST to the container's host ip.
```

Reply:
Done